### ISSUE TITLE
Symbol properties should be kept

### ISSUE THREAD

**Issue by ehmicky** [sindresorhus/filter-obj#21]
The `README` documents that symbol properties are not kept. However, this leads to some behavior that users might not expect:

```js
const symbol = Symbol('example')
const obj = { [symbol]: true, a: 1, b: 2 }
const objCopy = excludeKeys(obj, ['b'])
objCopy[symbol] // undefined. Unexpected: the consumer only wanted to exclude 'b'

const objCopyTwo = includeKeys(obj, [symbol, 'a'])
objCopyTwo[symbol] // undefined. Unexpected: the consumer intended to keep symbol
```

I can submit a PR.

**Comment by sindresorhus** [comment]
It makes sense to me to copy over symbols.

**Comment by ehmicky** [comment]
Fixed in #23.
